### PR TITLE
[ES] Add `G` missing stubs (4)

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

Missing G stubs from the `Missing stubs (up-to-date in English)` section according to the osu! wiki status in spanish.

28 stubs in total, divided into 4 pull requests of 7 stubs each

### CODE REVIEW

File: wiki/Gameplay/Spectating/es.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+stub: true
+tags:
+  - spec
+  - spectate
+  - spectator
+  - observar
+  - espectador
+---
+
+# Espectador
```
Comment (agatemosu) [inline wiki/Gameplay/Spectating/es.md:11]: Cámbialo en `Gameplay` también
```suggestion
# Observación
```

File: wiki/Gameplay/Spectating/es.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+**Espectar** (a menudo abreviado como «spec») es un término usado para describir el acto de observar a otro jugador jugando un [beatmap](/wiki/Beatmap) en tiempo real (con aproximadamente 500 ms de retraso). Para observar, abre el chat extendido y haz clic en un panel de usuario gris.
```
Comment (agatemosu) [inline wiki/Gameplay/Spectating/es.md:13]:
```suggestion
**Observar** o **spectate** (a menudo abreviado como «spec»), es un término usado para describir el acto de observar a otro jugador jugando un [beatmap](/wiki/Beatmap) en tiempo real (con aproximadamente 500 ms de retraso). Para observar, abre el chat extendido y haz clic en un panel de usuario gris.
```

File: wiki/Gameplay/Tablet_cover/es.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+## Bricolaje vs. fabricada
+
+Muchas publicaciones en los foros que preguntan dónde comprar o fabricar cubiertas para tabletas a menudo mencionan lo fácil que es crear una. Las cubiertas comunes para tabletas de bricolaje usan una variedad de materiales que se pueden encontrar en casa o en las tiendas a un precio relativamente económico. Mientras que las cubiertas para tabletas fabricadas suelen usar materiales especializados para reducir el desgaste de las puntas y durar más.
+
+### Bricolaje
+
+Muchos jugadores a menudo afirman que las cubiertas para tabletas hechas por uno mismo «hacen el trabajo» y, por lo tanto, son favorables debido a su bajo costo y accesibilidad. Las cubiertas para tabletas de bricolaje suelen usar materiales que se pueden encontrar en la mayoría de los hogares y, a menudo, se sujetan con cinta adhesiva o algún objeto pesado.
+
+Los materiales comunes para cubiertas de tabletas de bricolaje suelen usar uno o más de los siguientes materiales:
```
Comment (agatemosu) [inline wiki/Gameplay/Tablet_cover/es.md:29]: "tableta de bricolaje" es muy extraño, "tabletas hechas por ti" o algo así sería mejor

File: wiki/Gameplay/Tablet_cover/es.md [hunk 4]
```diff
@@ -18,15 +18,15 @@ Una **cubierta para la tableta** es una lámina de material protector especial q
 
 Las cubiertas para tabletas suelen tener una forma casi idéntica a la de los protectores de pantalla de los teléfonos inteligentes, excepto por el hecho de que el lado que se pega solo usa una tira de material adhesivo a lo largo de los bordes exteriores del material protector. Esto es para evitar daños con el material de la superficie real de la tableta y permite el uso de la sensibilidad a presión y otras funciones para el dibujo digital.
 
-## Bricolaje vs. fabricada
+## Casera vs. fabricada
```
Comment (agatemosu) [inline wiki/Gameplay/Tablet_cover/es.md:21]:
```suggestion
## Caseras vs. fabricadas
```

File: wiki/Gameplay/Tablet_cover/es.md [hunk 5]
```diff
@@ -18,15 +18,15 @@ Una **cubierta para la tableta** es una lámina de material protector especial q
...
 
-Muchas publicaciones en los foros que preguntan dónde comprar o fabricar cubiertas para tabletas a menudo mencionan lo fácil que es crear una. Las cubiertas comunes para tabletas de bricolaje usan una variedad de materiales que se pueden encontrar en casa o en las tiendas a un precio relativamente económico. Mientras que las cubiertas para tabletas fabricadas suelen usar materiales especializados para reducir el desgaste de las puntas y durar más.
+Muchas publicaciones en los foros que preguntan dónde comprar o fabricar cubiertas para tabletas a menudo mencionan lo fácil que es crear una. Las cubiertas caseras comunes para tabletas usan una variedad de materiales que se pueden encontrar en casa o en las tiendas a un precio relativamente económico. Mientras que las cubiertas para tabletas fabricadas suelen usar materiales especializados para reducir el desgaste de las puntas y durar más.
 
-### Bricolaje
+### Casera
```
Comment (agatemosu) [inline wiki/Gameplay/Tablet_cover/es.md:25]:
```suggestion
### Caseras
```

File: wiki/Gameplay/Tablet_cover/es.md [hunk 6]
```diff
@@ -18,15 +18,15 @@ Una **cubierta para la tableta** es una lámina de material protector especial q
...
 
-Muchos jugadores a menudo afirman que las cubiertas para tabletas hechas por uno mismo «hacen el trabajo» y, por lo tanto, son favorables debido a su bajo costo y accesibilidad. Las cubiertas para tabletas de bricolaje suelen usar materiales que se pueden encontrar en la mayoría de los hogares y, a menudo, se sujetan con cinta adhesiva o algún objeto pesado.
+Muchos jugadores a menudo afirman que las cubiertas para tabletas hechas por uno mismo «hacen el trabajo» y, por lo tanto, son favorables debido a su bajo costo y accesibilidad. Las cubiertas caseras para tabletas suelen usar materiales que se pueden encontrar en la mayoría de los hogares y, a menudo, se sujetan con cinta adhesiva o algún objeto pesado.
 
-Los materiales comunes para cubiertas de tabletas de bricolaje suelen usar uno o más de los siguientes materiales:
+Los materiales comunes para las cubiertas caseras de las tabletas suelen usar uno o más de los siguientes materiales:
```
Comment (agatemosu) [inline wiki/Gameplay/Tablet_cover/es.md:29]:
```suggestion
Los materiales comunes de las cubiertas caseras para tabletas suelen usar uno o más de los siguientes materiales:
```

File: wiki/Gameplay/Tablet_cover/es.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+Muchos jugadores a menudo afirman que las cubiertas para tabletas hechas por uno mismo «hacen el trabajo» y, por lo tanto, son favorables debido a su bajo costo y accesibilidad. Las cubiertas caseras para tabletas suelen usar materiales que se pueden encontrar en la mayoría de los hogares y, a menudo, se sujetan con cinta adhesiva o algún objeto pesado.
+
+Los materiales comunes para las cubiertas caseras de las tabletas suelen usar uno o más de los siguientes materiales:
+
+- Libros
+- Papel laminado
+- Tiras de papel
+- Protectores de pantalla de cristal
+
+### Fabricada
```
Comment (agatemosu) [inline wiki/Gameplay/Tablet_cover/es.md:36]:
```suggestion
### Fabricadas
```

File: wiki/Gameplay/Tablet_cover/es.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
+## Casera vs. fabricada
+
+Muchas publicaciones en los foros que preguntan dónde comprar o fabricar cubiertas para tabletas a menudo mencionan lo fácil que es crear una. Las cubiertas caseras comunes para tabletas usan una variedad de materiales que se pueden encontrar en casa o en las tiendas a un precio relativamente económico. Mientras que las cubiertas para tabletas fabricadas suelen usar materiales especializados para reducir el desgaste de las puntas y durar más.
+
+### Casera
+
+Muchos jugadores a menudo afirman que las cubiertas para tabletas hechas por uno mismo «hacen el trabajo» y, por lo tanto, son favorables debido a su bajo costo y accesibilidad. Las cubiertas caseras para tabletas suelen usar materiales que se pueden encontrar en la mayoría de los hogares y, a menudo, se sujetan con cinta adhesiva o algún objeto pesado.
```
Comment (agatemosu) [inline wiki/Gameplay/Tablet_cover/es.md:27]:
```suggestion
Muchos jugadores a menudo afirman que las cubiertas caseras para tabletas «hacen el trabajo» y, por lo tanto, son favorables debido a su bajo costo y accesibilidad. Las cubiertas caseras para tabletas suelen usar materiales que se pueden encontrar en la mayoría de los hogares y, a menudo, se sujetan con cinta adhesiva o algún objeto pesado.
```

**Review by TicClick — APPROVED** [review]

**Pull request merged**